automation/src/test/scala/org/broadinstitute/dsde/test/api/BillingProjectOwnerPermissionSpec.scala

```diff
@@ -0,0 +1,270 @@
+package org.broadinstitute.dsde.test.api
+
+import org.broadinstitute.dsde.test.util.AuthDomainMatcher
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.{Credentials, UserPool}
+import org.broadinstitute.dsde.workbench.fixture.{BillingFixtures, GroupFixtures, WorkspaceFixtures}
+import org.broadinstitute.dsde.workbench.service.BillingProject.BillingProjectRole
+import org.broadinstitute.dsde.workbench.service.Orchestration.groups.GroupRole
+import org.broadinstitute.dsde.workbench.service.{Rawls, RestException, Sam}
+import org.scalatest.{FreeSpec, Matchers}
+
+import scala.util.Try
+
+class BillingProjectOwnerPermissionSpec extends FreeSpec with WorkspaceFixtures with BillingFixtures with GroupFixtures  with Matchers {
+
+  /*
+ * Unless otherwise declared, this auth token will be used for API calls.
+ * We are using a curator to prevent collisions with users in tests (who are Students and AuthDomainUsers), not
+ *  because we specifically need a curator.
+ */
+
+  val defaultUser: Credentials = UserPool.chooseCurator
+  val authTokenDefault: AuthToken = defaultUser.makeAuthToken()
+
+
+  "removing permissions from billing project owners for workspaces with auth domains" - {
+
+    "+ project owner, + group member, create workspace, - group member" in {
+      val owner = UserPool.chooseProjectOwner
+      val creator = UserPool.chooseStudent
+      val user = owner
+      implicit val token: AuthToken = creator.makeAuthToken()
+
+      withCleanBillingProject(owner, List(creator.email)) { projectName =>
+        withGroup("group", List(user.email)) { groupName =>
+          withWorkspace(projectName, "GroupApiSpec_workspace", Set(groupName)) { workspaceName =>
+            AuthDomainMatcher.checkVisibleAndAccessible(projectName, workspaceName, List(groupName))(user.makeAuthToken())
+            Sam.user.removeUserFromPolicy(groupName, GroupRole.Member.toString, user.email)
+            AuthDomainMatcher.checkVisibleNotAccessible(projectName, workspaceName)(user.makeAuthToken())
+          }
+        }
+      }
+    }
+
+    "+ project owner, + group member, create workspace, - project owner" in {
+      val owner = UserPool.chooseProjectOwner
+      val creator = owner
```

Comment:
`owner` isn't actually used anywhere. Take out `val creater = owner` and just have `val creator = UserPool.chooseProjectOwner`

Reply:
updated here and in other tests.